### ISSUE TITLE
[Settings]: ‘Columns’ and ‘Rows’ edit box are the editable spin box, but same info is not provided to Screen Reader users.

### ISSUE THREAD

**Issue by v-rpundir** [microsoft/terminal#12008]
### Windows Terminal version

1.12.3472.0

### Windows build number

10.0.22504.1010

### Other Software

**Test Environment:**
OS: Windows 11 Version Dev (OS Build 22504.1010)
App: Windows Terminal Preview
Screen Reader: Narrator
Tool: Accessibility Insights for Windows Version 1.1.1741.1

### Steps to reproduce

**Repro Steps:**
1. Open Windows Terminal.
2. Open Settings page using 'Ctr+,'.
3. Open Narrator using 'Win+Ctrl+ Enter' key. Turn OFF Narrator scan mode using 'Caps lock + Space' key.
4. Activate 'Startup' tab and then navigate to 'Columns' edit field then observe the issue.

**User Experience:**
Screen Reader users will be impacted here as they are not able to know about the spin property of edit control, which will not enhance the UX of those users.

**Guideline Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

**Attachments:**
[‘Columns’ and ‘Rows’ edit box are the editable spin box, but same info is not provided to Screen Reader users..zip](https://github.com/microsoft/terminal/files/7750603/Columns.and.Rows.edit.box.are.the.editable.spin.box.but.same.info.is.not.provided.to.Screen.Reader.users.zip)

### Expected Behavior

 Screen Reader should give some hint about ‘Columns’ and ‘Rows’ edit box 'spin' property.
i.e. For Column with value 120 Screen Reader should announce something like this 'Columns Spinner edit 120' OR 'Columns edit 120 Use up or down arrow keys to increase or decrease count'

### Actual Behavior

 ‘Columns’ and ‘Rows’ edit box are the editable spin box, but same info is not provided to Screen Reader users. _Issue repro with all spinner edit fields._
i.e. For Column with value 120 Screen Reader is announcing 'Column edit 120'.
